variables.tf

```diff
@@ -25,6 +25,12 @@ variable "prefix" {
 }
 
 
+variable "rds_security_group_id" {
+  type        = string
+  description = "Cumulus' RDS Security Group's ID."
+}
```

Comment:
should it have a default value?

Reply:
Nope. We need Cumulus to give us this, or we can't reach the DB.